lib/note-content-editor.jsx

```diff
@@ -1,11 +1,41 @@
 import React, { PropTypes } from 'react';
-import { Editor, EditorState, ContentState } from 'draft-js';
+import {
+	Editor, EditorState, ContentState, Modifier, SelectionState
+} from 'draft-js';
 import { invoke, noop } from 'lodash';
 
 function plainTextContent( editorState ) {
 	return editorState.getCurrentContent().getPlainText( '\n' )
 }
 
+const EditorStateModifier = {
+	insertText( editorState, selection, characters ) {
+		return EditorState.push(
+			editorState,
+			Modifier.insertText(
+				editorState.getCurrentContent(), selection, characters
+			),
+			'insert-characters'
+		);
+	},
+
+	removeRange( editorState, selection = editorState.getSelection() ) {
+		return EditorState.push(
+			editorState,
+			Modifier.removeRange( editorState.getCurrentContent(), selection ),
+			'remove-range'
+		);
+	},
+
+	splitBlock( editorState, selection = editorState.getSelection() ) {
```

Comment:
hmm… good for consistency but also a little confusing since we're using the same name for a different function.

why not simply use `splitBlock` as it comes? shouldn't this get a different name to indicate that it does something different?